script/configure-assets.js

```diff
@@ -45,6 +45,19 @@ function addAssetHashes() {
         });
       }
     });
+
+    // Create a copy of the proxy-write files without cache-bust hashes
+    [
+      'proxy-rewrite.entry.js',
+      'styleConsolidated.css',
+      'static-pages.css',
+      'vendor.entry.js',
+    ].forEach(unhashedName => {
+      const hashedName = manifest[unhashedName];
+
+      files[`generated/${unhashedName}`] = files[hashedName.substr(1)]; // eslint-disable-line no-param-reassign
+    });
+
```

Comment:
Ah, based on how you had to check for `fileData` in the other middleware, I think it expects you to clone it like -

```js
const path = `generated/${unhashedName}`;
const copyFile = files[hashedName.substr(1)];
files[path] = {
   path,
   contents: new Buffer(copyFile.contents.toString())
};
```

My bad, I always forget that part. Clone like this should save us from having to touch more than one file.

Reply:
@ncksllvn I just tried removing it and the original script without the fileData check in `check-redirects`  results in a successful build so I think we are okay. I'll remove that extra safety check in the next PR if there are no side effects